app/page.tsx

```diff
@@ -66,13 +68,58 @@ function AboutUsOverview() {
   );
 }
 
+function SponsorsPreview() {
+  return (
+    <Box mx='auto' textAlign='center' width='100%' mb={10} justifyContent='center'>
+      <Box sx={{ display: 'flex', justifyContent: 'center', width: '100%' }}>
```

Comment:
You should also be able to remove this outer Box

Reply:
left this in because it makes sure the line doesn't stay fixed at 1000px.
without the outer box:
<img width="425" alt="image" src="https://github.com/UNSW-WIESoc/website/assets/161536467/6479411f-14fa-4a64-8ffd-2090a987bc90">